mobile/apps/artwork/components/bid/templates/bid_button.jade

```diff
@@ -1,9 +1,36 @@
+- auction = artwork.auction
+- bidderStatus = helpers.getBidderStatus(me, auction)
+- actionUrl = helpers.getRedirectActionUrl(bidderStatus, artwork, auction)
+
 form.artwork-auction-bid-form.js-artwork-auction-bid-form(
-    method="GET"
-    action= "/auction/#{artwork.auction.id}/bid/#{artwork.id}"
-  )
-    .artwork-auction-bid-module__bid-button
-      button.auction-avant-garde-black-button(
-        type= 'submit'
-        disabled= false
-      ) Bid
+  method="GET"
+  action="#{actionUrl}"
+)
+  .artwork-auction-bid-module__bid-button
```

Comment:
Is there any way we can reuse this component on mobile and desktop? It's a lot of view logic that seems like could benefit from some sharing.